xrpl/models/requests/ledger_entry.py

```diff
@@ -153,19 +155,101 @@ class RippleState(BaseModel):
 @require_kwargs_on_init
 @dataclass(frozen=True)
 class Ticket(BaseModel):
+    """Required fields for requesting a Ticket if not querying by object ID."""
+
+    owner: str = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    ticket_sequence: int = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class XChainClaimID(BaseModel):
+    """Required fields for requesting an XChainClaimID if not querying by object ID."""
+
+    locking_chain_door: str = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    locking_chain_issue: Currency = REQUIRED  # type: ignore
     """
-    Required fields for requesting a Ticket, if not querying by
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    issuing_chain_door: str = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    issuing_chain_issue: Currency = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    xchain_claim_id: Union[int, str] = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
```

Comment:
Can you add the description of these fields? Or are you waiting for docs to be done?